nile-backend/src/test/java/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/NileTest
```

Comment:
Is this file necessary? I put this information in the application-test.properties.

Reply:
yeah, my bad. Left that in there when I was testing before the one you added was in. I deleted it